.github/workflows/native-build-development.yml

```diff
@@ -43,21 +43,40 @@ jobs:
       - name: Check RSS
         env:
           QUICKSTART_VERSION: "1.0-SNAPSHOT"
-          RSS_THRESHOLD: "25000"
+          RSS_THRESHOLD: "40000"
           START_TIME_THRESHOLD: "1000"
+          BUCKET_NAME: ${{ secrets.BUCKET_NAME }}
+          REMOTE_OBJECT_NAME: "pmap.out"
+          API_KEY: ${{ secrets.API_KEY }}
+          SERVICE_INSTANCE_ID: ${{ secrets.SERVICE_INSTANCE_ID }}
+
         run: |
-          getting-started/target/getting-started-${QUICKSTART_VERSION}-runner -Xmx2m > server.log &
+          sudo apt-get update -o Dir::Etc::sourcelist="sources.list" -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
+          sudo apt-get install -y gnupg2 gnupg-agent
+          echo Installing SDKMAN
+          curl -s "https://get.sdkman.io" | bash
+          source ~/.sdkman/bin/sdkman-init.sh
+          sdk install jbang
```

Comment:
You really want to make Max happy, right?

Reply:
Just finding it useful :)

Reply:
fyi, you could have used the maxandersen/jbang-action docker image here